MANIFEST.in

```diff
@@ -5,7 +5,7 @@ include ez_setup.py
 include ah_bootstrap.py
 include setup.cfg
 
-recursive-include *.pyx *.c *.pxd
+global-include *.pyx *.c *.pxd
```

Comment:
I think the problem here is this really should be `recursive-include <name of package> *.pyx *.c *.pxd`